database/seeds/items.js

```diff
@@ -0,0 +1,17 @@
+const items = require("../../items.js");
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("items")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("items").insert([
+        { ...items.stick },
+        { ...items.oldSword },
+        { ...items.oldShield },
+        { ...items.magicSword },
+        { ...items.ultimateSpear }
```

Comment:
You could do this insert in a bit more programmatic way:
```javascript
  return knex('items').insert(
    Object.keys(items).map(
      itemName => ({ ...items[itemName] })
    )
  );
```
this'll help out if you ever add more items (or even remove any) so that you don't have to worry about updating the same stuff in multiple files rather than one.